src/components/tools/drawing-tool/drawing-toolbar.tsx

```diff
@@ -26,9 +28,11 @@ const kClosedPalettesState = { showStamps: false, showStroke: false, showFill: f
 interface IProps extends IFloatingToolbarProps, IRegisterToolApiProps {
   model: ToolTileModelType;
 }
+
 export const ToolbarView: React.FC<IProps> = (
               { documentContent, model, onIsEnabled, ...others }: IProps) => {
   const drawingContent = model.content as DrawingContentModelType;
+  const toolbarButtonSetting = useSettingFromStores("tools", "drawing") as unknown as string[];
```

Comment:
```suggestion
  const toolbarButtonSetting = useSettingFromStores("tools", "drawing") as unknown as string[] | undefined;
```
If the setting doesn't exist `undefined` will be returned and the types should acknowledge that -- and the code below should deal with it.